### ISSUE TITLE
Typo in JsoupResponse javadoc

### ISSUE THREAD

**Issue by php-coder** [jcabi/jcabi-http#147]
When I'm opening [`JsoupResponse` javadoc](http://http.jcabi.com/apidocs-1.16/com/jcabi/http/response/JsoupResponse.html) I see this:
```
Implicit tags will be made explicit (e.g. a naked <td> will be wrapped in a <table><trgt;<td>?) 
```
Here, instead of `<trgt;` should be `<tr>`.

**Comment by dmarkov** [comment]
@yegor256 please dispatch this issue

**Comment by yegor256** [comment]
@php-coder fixed in e93351c thanks!
